test/test_analysis.py

```diff
@@ -51,6 +52,39 @@ async def test_add_cram(self):
         self.assertEqual(1, analysis_samples[0]['sample_id'])
         self.assertEqual(analyses[0]['id'], analysis_samples[0]['analysis_id'])
 
+    @run_as_sync
+    async def test_get_analysis(self):
+        """
+        Test adding an analysis of type ANALYSIS_RUNNER
+        """
+
+        await self.al.insert_analysis(
+            analysis_type=AnalysisType.ANALYSIS_RUNNER,
+            status=AnalysisStatus.IN_PROGRESS,
```

Comment:
This more closely aligns with what the cloud function reports, but shouldn't be a big deal either way
```suggestion
            status=AnalysisStatus.UNKNOWN,
```